cpp/include/cudf/types.hpp

```diff
@@ -137,6 +137,15 @@ enum class nan_policy : bool {
   NAN_IS_VALID  ///< treat nans as valid elements (non-null)
 };
 
+/**
+ * @brief Enum to consider different elements (of floating point types) holding NaN value as equal
+ * or unequal
+ */
+enum class nan_equality /*unspecified*/ {
```

Comment:
Should specify as `bool` now only two possible values are determined?

Reply:
Not necessary. That just complicates things if we add another option later.